dodona-prolog.dockerfile

```
@@ -1,4 +1,4 @@
-FROM swipl:8.2.2
+FROM swipl:8.3.18
```

```suggestion
FROM swipl:8.2.4
```
Use latest stable instead of the development branch